src/extension/options-page/Drawer.tsx

```diff
@@ -62,9 +62,7 @@ const useStyles = makeStyles(theme => ({
     masktext: {
         display: 'block',
         marginTop: -20,
-        marginBottom: -20,
         width: 120,
-        height: 60,
```

Comment:
These should have been fixed in master branch.

Reply:
Revoked!